spec/models/application_spec.rb

```diff
@@ -59,10 +59,9 @@
 
     it 'removes the flag if the value is 0' do
       flag = flags.sample.to_sym
-      subject.flags = [flag]
```

Comment:
Thanks, a few things that concern me with this:

* I wasn't quite sure if `flags=` was ever accessed directly.
* Not sure how these were stored in production with AR 4.1, if they were strings or symbols.
* From what I could gather is using the `flags=` method AR stores keeps the instance as a symbol, then saves it as a string.

Anyway the doesn't give instil confidence in it and the code might be doing unpredictable things. I can take another crack at it.